api/core/tools/provider/builtin/stablediffusion/tools/stable_diffusion.py

```diff
@@ -20,7 +20,7 @@
     "subseed": -1,
     "subseed_strength": 0,
     "seed_resize_from_h": -1,
-    'sampler_index': 'DPM++ SDE Karras',
+    # 'sampler_index': 'DPM++ SDE Karras',
```

Comment:
what about introduce a new parameter named `samlper_index` as different models depend on different sampler.

Reply:
ok,I will turn it into a parameter and look forward to the frontend team making it a configurable value.